src/components/views/styl/Metric.styl

```diff
@@ -57,20 +60,52 @@ canvas {
 }
 
 #metric-details {
+    margin-bottom: 20px;
+
     dl {
-        display: flex;
+        display: grid;
         font-size: .8em;
+        font-size: 12px;
+        grid-gap: $legend-grid-gap;
+        grid-template-columns: 160px 160px 160px;
```

Comment:
Alternatively, `1fr 1fr 1fr`. That is, assuming they should all have equal widths and should collectively take up `100%` of their parent's width.

Reply:
`fr` is a new unit for use with CSS grid. It represents one fraction of space. So `1fr 1fr 1fr` means three columns of 33% width. `1fr 1fr` is two columns of 50% width. Etc.

Reply:
I did overlook that though when I set it with 33% had some box-sizing issues due to padding. Might be good to set to border-box and proceed with `fr` indeed.

Reply:
Yeah `fr` is nice, especially when later making things responsive.